### ISSUE TITLE
Related to position & mass: adjust the range of values that describe the size of the arrows

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#114]
The size descriptions for the force arrows seem a bit off to me.

When everything is working properly, we may need to tweak the ranges in the model that are producing the size descriptions for the arrows.

My understanding is that force changes affected by position are more dramatic - like in the PLOTTED curve in the GFL Regular design doc in the [Sim Model](https://docs.google.com/document/d/1-37qAgde2XrlXBQae2SgjartM35_EnzDD9pdtd3nXAM/edit#heading=h.gehkpbiilqw) section.

**Comment by terracoda** [comment]
@zepumph, could you tell me how the model is handling the size of the force arrows?
For example, what are the relationships between Force and Position and Force and Mass.

For implementation should I just decide what force value range maps to the visual size I think should be described as :
- tiny
- very small
- small
- medium size
- large
- very large
- huge

This is a 7-point scale whereas described distance has 9 described regions that range in actual size.
I'm just wondering if the size changes should coincide at all with the described position ranges, or are the 2 sets of descriptions irrelevant?

**Comment by zepumph** [comment]
@terracoda here is what the logic looks like in REGULAR:
![image](https://user-images.githubusercontent.com/6856943/56152427-1b3d1d80-5f60-11e9-87a2-68e89df152c3.png)

In BASICS it uses a linear function based on the range of the force:
```js
  var MIN_FORCE = 35; // N
  var MAX_FORCE = 1070; // N
  const forceToIndex = new LinearFunction( MIN_FORCE, MAX_FORCE * 0.6, 0, 6, true );
```

That maps the range into the indices from 0-6. My guess is that the `0.6` multiplier was empirically determined such that everything above that value is the largest force string. 

Here are the current strings being used:
```js
  const SIZE_STRINGS = [
    tinyString,
    verySmallString,
    smallString,
    mediumSizeString,
    largeString,
    veryLargeString,
    hugeString
  ];
```


>I'm just wondering if the size changes should coincide at all with the described position ranges, or are the 2 sets of descriptions irrelevant?

I'm not sure what this question is asking. The Force descriptions are a third set of strings, dependent on the current position and mass.

In the model the forceProperty is derived from the object masses and the distance between the two in the following equation:

```js
new DerivedProperty( [
      this.object1.valueProperty, // see valueProperty in ISLCObject.js
      this.object2.valueProperty,
      this.object1.positionProperty,
      this.object2.positionProperty
    ], function( v1, v2, x1, x2 ) {
      var distance = Math.abs( x2 - x1 );
      return this.forceConstant * v1 * v2 / ( distance * distance );
     }
 ```

Here it is in the code. https://github.com/phetsims/inverse-square-law-common/blob/master/js/model/ISLCModel.js#L74

Let me know what else you would like to have to calculate these values.

**Comment by terracoda** [comment]
Thanks @zepumph.

**Comment by terracoda** [comment]
@zepumph and @emily-phet I made a table that shows the ranges visually and descriptively here:
[https://docs.google.com/document/d/1HdDG9ds2MdbCb21l9qk3cI8yBQxK6-wBWNXC4Tloji8/edit#heading=h.xv1hj7hfnqi8](https://docs.google.com/document/d/1HdDG9ds2MdbCb21l9qk3cI8yBQxK6-wBWNXC4Tloji8/edit#heading=h.xv1hj7hfnqi8)

Do they seem right to both of you?

**Comment by terracoda** [comment]
Notice I do not have the exact mathematical ranges as I can't reach them with the visual sim.

**Comment by emily-phet** [comment]
This looks reasonable to me. You might check with @kathy-phet and @ariel-phet to see if they have any suggestions for the wording. Specifically "tiny" stands out to me as being reasonable, but perhaps they might be able to weigh on whether or not their might be a discrepancy with how they would talk about that range of magnitude for force.  

I know the wording for those ranges have been around for awhile, so I don't think they need re-thinking, I'm just imagining a quick "sanity check" that they seem reasonable to folks who teach this content.

**Comment by zepumph** [comment]
I updated the regions to the table. In my opinion I still feel like the "tiny" region occupies too large a percentage of the "normal use"  play possibilities, but I understand why (because of the curve of the inverse square relationship). I think it is fine now. I'm also happy to tweak wording, just tell me when and how.

**Comment by terracoda** [comment]
@zepumph, what I put in the table was the current implementation when I made the changes.
I'll review before our meeting. I think we might need a hard visual maximum for tiny.

**Comment by terracoda** [comment]
@zepumph, I am deciding to leave the ranges as they are.
